cli/scaletest.go

```diff
@@ -955,16 +967,18 @@ func (r *RootCmd) scaletestWorkspaceTraffic() *clibase.Cmd {
 			th := harness.NewTestHarness(strategy.toStrategy(), cleanupStrategy.toStrategy())
 			for idx, ws := range workspaces {
 				var (
-					agentID uuid.UUID
-					name    = "workspace-traffic"
-					id      = strconv.Itoa(idx)
+					agentID   uuid.UUID
+					agentName string
```

Comment:
I didn't go through the whole PR, but why is it required to specify the agent name? isn't it always "main"?

Reply:
My understanding is that the agent name maps directly to what's specified in the workspace [Terraform](https://registry.terraform.io/providers/coder/coder/latest/docs/resources/agent), is this not the case?